### DISCUSSION TITLE
Commands syntax

### DISCUSSION THREAD

**Discussion by kassadGLA** [Zrips/CMI#5992] [Q&A]
Hello.
How to spawn a animal tamed by a specific player? Not only by command executor?

Regards.

**Comment by mrfloris** [comment]
As far as I know this isn't possible for a defined user, but spawning tamed is possible. From the docs: `cmi spawnmob horse:baby:tamed:n-Horsy` 
<img width="1046" alt="Screenshot 2021-09-08 at 09 26 19" src="https://user-images.githubusercontent.com/28841349/132465088-8b9bd9d3-681d-4a36-8328-52fa46bb6858.png">

I don't know if using `[playerName]` or `t:[playerName]` will do anything. 

If this is a custom command you can perhaps run it as a sudo command.

```
bperm:cmi.command.spawnmob! asConsole! cmi sudo PlayerName cmi spawnmob horse:tamed
```
Or something like that.
